### PR TITLE
Update tpl_shopping_cart_default.php

### PR DESCRIPTION
**Pull request by retched**
Hides the Update Cell in the Shopping Cart unless the option `SHOW_SHOPPING_CART_UPDATE` is specifically set to `Next to each Qty Box` or `Both Next to each Qty Box and Below all Products`.

### CODE REVIEW

**Comment by retched** [comment]
Running the shopping card through an evaluation tool suggested that to comply with WCAG standard 1.3.1 a `<TH>` tag shouldn't be empty or contain images with no alt text.

So I hide that column that would contain the update button UNLESS it is explicitly set in the Admin dashboard. As well as change the last column, the total column, to a regular `<TD>` instead of a `<TH>`.

**Comment by retched** [comment]
It was a suggestion that <th>'s shouldn't be empty, even with a non-breaking space, for accessibility. To quote the guideline I found:

https://webaim.org/techniques/tables/data#th
Table headers should never be empty. This is particularly of concern for the top-left cell of some tables

I'm not sure how stringent that is though.

========
Respectfully submitted,


Paul L. Williams III
________________________________

**Comment by drbyte** [comment]
One way we handle the "no empty header cell" elsewhere in Zen Cart is to simply give it a header of "Action". 
And often make that text aria-only.

**Comment by retched** [comment]
I didn't even consider that as a possibility. I can make a similar change in another commit.

**Comment by retched** [comment]
I reverted the empty cells to a `<th>` but gave each column an aria-label header with an empty cell. I also added two language definitions to `lang.zca_bootstrap_common.php` for this purpose.

File: includes/templates/bootstrap/templates/tpl_shopping_cart_default.php [hunk 1]
```diff
@@ -70,11 +70,11 @@
         <table id="shoppingCartDefault-cartTableDisplay" class="cartTableDisplay table table-bordered table-striped table-sm">
             <tr>
                 <th scope="col" id="cartTableDisplay-qtyHeading"><?php echo TABLE_HEADING_QUANTITY; ?></th>
-                <th scope="col" class="d-none d-sm-table-cell" id="cartTableDisplay-qtyUpdateHeading">&nbsp;</th>
+                <?php if (SHOW_SHOPPING_CART_UPDATE == 1 || SHOW_SHOPPING_CART_UPDATE == 3) { ?><th scope="col" class="d-none d-sm-table-cell" id="cartTableDisplay-qtyUpdateHeading"><span aria-label="<?php echo TEXT_CART_ARIA_HEADING_UPDATE_COLUMN; ?>">&nbsp;</span></th> <?php } ?>
```
Comment (lat9) [inline includes/templates/bootstrap/templates/tpl_shopping_cart_default.php:73]: I'm bit bit more persnickety and don't like the 'inline' PHP conditionals.  Please indent the `<php` conditional to line up with its parent and render the HTML portion separately (also indented to match the other `th` elements).

File: includes/templates/bootstrap/templates/tpl_shopping_cart_default.php [hunk 2]
```diff
@@ -108,7 +108,7 @@
 ?>
                     </div>
                 </td>
-                <td class="qtyUpdateCell text-center d-none d-sm-table-cell"><?php echo (!empty($product['buttonUpdate'])) ? $product['buttonUpdate'] : ''; ?></td>
+                <?php if (SHOW_SHOPPING_CART_UPDATE == 1 || SHOW_SHOPPING_CART_UPDATE == 3) { ?><td class="qtyUpdateCell text-center d-none d-sm-table-cell"><?php echo (!empty($product['buttonUpdate'])) ? $product['buttonUpdate'] : ''; ?></td><?php } ?>
```
Comment (lat9) [inline includes/templates/bootstrap/templates/tpl_shopping_cart_default.php:111]: Ditto for the `<td>`.

**Review by lat9 — CHANGES_REQUESTED** [review]
Resolve those couple of indentation 'issues' and it's good to go.

**Comment by retched** [comment]
Should be done. Also went ahead and adjusted the quantity field to render the HTML aligned within the `<td>` itself too.
image

**Comment by retched** [comment]
Squashed the three commits to make things easier to follow.

**Review by lat9 — APPROVED** [review]

**Comment by lat9** [comment]
Tie this PR to the associated issue, #410.

**Pull request merged**